src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newSource = sourceString
+    .split(';')
+    .map((sourceString) => sourceString.trim())
+    .filter((sourceString) => sourceString.length > 0);
+
+  const newLine = newSource.map((line) => line.split(':'));
+
+  newLine.sort();
+
+  const styleObject = Object.fromEntries(newLine);
```

Comment:
Ensure that the input string is always in the expected format. Consider adding error handling for cases where the input string might not contain colons or semicolons, which could lead to unexpected results.